qa/suites/upgrade/pacific-x/stress-split/2-first-half-tasks/rbd-cls.yaml

```diff
@@ -3,7 +3,7 @@ meta:
    run basic cls tests for rbd
 first-half-tasks:
 - workunit:
-    branch: octopus
+    branch: reef
```

Comment:
Same question as in https://github.com/ceph/ceph/pull/50747#discussion_r1153810272.  There is an obvious inconsistency with https://github.com/ceph/ceph/pull/50759.